lib/puppet/face/module/list.rb

```diff
@@ -17,6 +17,10 @@
       summary _("Whether to show dependencies as a tree view")
     end
 
+    option '--strict-semver' do
+      summary _('Whether version ranges should exclude pre-release versions')
+    end
+
```

Comment:
While I agree that setting defaults closer to a parameter declaration is a good thing, all module sub-command defaults are currently setting defaults in `Puppet::ModuleTool.set_option_defaults options`. I continued to use that pattern here. Changing that pattern deserves a ticket of its own I think.